connectors/src/connectors/google_drive/temporal/spreadsheets.ts

```diff
@@ -496,3 +511,16 @@ export async function deleteSpreadsheet(
     await deleteAllSheets(connector, sheetsInSpreadsheet, file);
   }
 }
+
+function isGAxiosServiceUnavailablError(err: unknown): boolean {
+  return (
+    err instanceof Error &&
+    "code" in err &&
+    err.code === 503 &&
```

Comment:
Given that 503 means Service unavailable, should we keep this typeguard a bit more flexible?